### PR TITLE
Update exporters.md for inclusion of Typescript code snippet

### PR DESCRIPTION
**Pull request by zeus2611**
Added Typescript code snippet alongside javascript for exporters.

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
Thanks!

File: content/en/docs/instrumentation/js/exporters.md [hunk 1]
```diff
@@ -26,7 +26,30 @@ update `tracing.ts|js` from the
 [Getting Started](/docs/instrumentation/js/getting-started/nodejs/) like the
 following:
 
-```js
+{{< tabpane langEqualsHeader=true >}}
+{{< tab Typescript >}}
+/*tracing.ts*/
```
Comment (pichlermarc) [inline content/en/docs/instrumentation/js/exporters.md:31]: Looks like an import is missing here :slightly_smiling_face: 
```suggestion
/*tracing.ts*/
import * as opentelemetry from '@opentelemetry/sdk-node';
```

File: content/en/docs/instrumentation/js/exporters.md [hunk 2]
```diff
@@ -190,12 +215,22 @@ npm install --save @opentelemetry/exporter-zipkin
 Update your opentelemetry configuration to use the exporter and to send data to
 your zipkin backend:
 
-```javascript
+{{< tabpane langEqualsHeader=true >}}
+{{< tab Typescript >}}
+import { ZipkinExporter } from "@opentelemetry/exporter-zipkin");
```
Comment (pichlermarc) [inline content/en/docs/instrumentation/js/exporters.md:220]:
```suggestion
import { ZipkinExporter } from "@opentelemetry/exporter-zipkin";
```

**Review by pichlermarc — COMMENTED** [review]
Thanks for adding this! :slightly_smiling_face:

**Review by pichlermarc — APPROVED** [review]

**Pull request merged**